.github/workflows/ci.yml

```
@@ -183,3 +191,15 @@ jobs:
           TAG_NAME: ${{ github.ref_name }}
         run: |
           gh release edit "$TAG_NAME" --notes-file doc/manual/source/release-notes-determinate/"$TAG_NAME".md || true
+
+  mirror:
+    if: (!github.repository.fork && (startsWith(github.ref, 'refs/tags/')))
```

actionlint (1.7.12)

[error] receiver of object dereference "fork" must be type of object but got "string"

(expression)